src/go/k8s/pkg/resources/resource.go

```diff
@@ -56,6 +56,8 @@ func CreateIfNotExists(
 	return false, nil
 }
 
+const debugLevel = 10
```

Comment:
Nit: According to https://github.com/go-logr/logr#how-do-i-choose-my-v-levels, 10 is the least-priority level, which would be equivalent to `trace`.